### DISCUSSION TITLE
How to assign a relationship when user saves the form

### DISCUSSION THREAD

**Discussion by oitkenmen** [Budibase/budibase#5775] [Show and tell]
Hello. This is just a short description of how to assign a relationship when the user saves the form. For future reference. I tried to find some information about this and finally figured out:

1. For example, there's a form. The user fills the form and clicks "Save".
2. The "Save" button has an action called "Save Row".
3. Inside this action, we need to add a Column (our relationship picker).
4. In Value, we need to add `{{ Current User._id }}`

**Comment by melohagan** [comment]
Hi @berkimbayev 

Thanks for sharing!

Here's a few other related docs pages that may be useful for further reading:

- <a href="https://docs.budibase.com/docs/referencing">Action Referencing</a>
- <a href="https://docs.budibase.com/docs/users-table">Users Table</a>
- <a href="https://docs.budibase.com/docs/form-steps">Form Steps</a>
